operations/mimir/autoscaling.libsonnet

```diff
@@ -147,7 +150,7 @@
     ],
   }),
 
-  local newQueryFrontendScaledObject(name, cpu_requests, memory_requests, min_replicas, max_replicas) = self.newScaledObject(
+  local newQueryFrontendScaledObject(name, cpu_requests, memory_requests, min_replicas, max_replicas, memory_coefficient) = self.newScaledObject(
```

Comment:
I would rather call it "target utilization" instead of "memory coefficient", to also keep it consistent with the naming we have in "automated scaling". The concept is the same: target to an utilization lower than 100% the resources we request.

Reply:
That sounds good to me @pracucci - I will make the change.

Reply:
Done.

Reply:
BTW, did you mean "memory target utilization"? Otherwise it's not clear if it's CPU or memory the parameter is for.

Reply:
> did you mean "memory target utilization"?

Yes!